data/pc/21w07a/protocol.json

```diff
@@ -3920,6 +3944,37 @@
             }
           ]
         ],
+        "packet_sculk_vibration_signal": [
+          "container",
+          [
+            {
+              "name": "sourcePosition",
+              "type": "position"
+            },
+            {
+              "name": "destinationIdentifier",
+              "type": "string"
+            },
+            {
+              "name": "destination",
+              "type": [
+                "switch",
+                {
+                  "compareTo": "destinationIdentifier",
+                  "fields": {
+                    "block": "i64",
```

Comment:
this doesn't seem good
what is that long?

Reply:
https://wiki.vg/Pre-release_protocol#Sculk_Vibration_Signal

Reply:
yes wiki.vg is missing the information too.
I bet it's a position https://wiki.vg/Protocol#Data_types

Reply:
added